bin/github-markup

```diff
@@ -3,8 +3,26 @@
 $LOAD_PATH.unshift File.dirname(__FILE__) + "/../lib"
 require 'github/markup'
 
-if ARGV[0] && File.exists?(file = ARGV[0])
-  puts GitHub::Markup.render(file)
-else
-  puts "usage: #$0 FILE"
+if ARGV.size < 1
+  print "usage: #$0 FILE [ FILES ... ]\n"
+  exit
 end
+
+sources = []
+
+ARGV.each { |s|
+  begin
+    file = File.open( s, "r" )
+    sources.push [ s, file ]
+  rescue Exception => e
+    print "error: #{e.message}\n"
```

Comment:
I think it would be acceptable to print this to `STDERR` then exit with an error code. Currently this will go straight to `STDOUT` with the result of the output of the command.